The context stub misses " around the arguments to protect spaces in the args
---
 scripts/context/stubs/unix/context |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: context-2008.04.18/scripts/context/stubs/unix/context
===================================================================
--- context-2008.04.18.orig/scripts/context/stubs/unix/context	2008-04-19 11:01:13.000000000 +0200
+++ context-2008.04.18/scripts/context/stubs/unix/context	2008-04-19 11:01:20.000000000 +0200
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-mtxrun --script context $@
+mtxrun --script context "$@"
